src/registry.rs

```diff
@@ -171,6 +171,7 @@ pub struct NpmPackageVersionInfo {
   #[serde(default)]
   #[serde(deserialize_with = "deserializers::hashmap")]
   pub scripts: HashMap<String, String>,
+  pub deprecated: Option<String>,
```

Comment:
@dsherret suggested we do a custom deserializer, similar to `deserializers::hashmap`, as it's not guaranteed that this value is `null` or `string`.

Reply:
Yeah, it would be good to add a test for this too (ex. `deprecated` being an object or array should deserialize to `None`).

Reply:
I updated the code with your suggestion.
@bartlomieju Could you please take a look?